### PR TITLE
Added example frontend and backend using the react-slack-login

### PR DESCRIPTION
**Pull request by MrKrrot**

### CODE REVIEW

**Comment by hectorbenitez** [comment]
@MrKrrot
- Readme is missing
- Package.json + lock is missing

**Comment by MrKrrot** [comment]
Added 
INSTALLATION.md
.env.example

And modified the client and server to work in the same port instead different ports (3000 & 5000 for example).

File: INSTALLATION.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+| VITE_REDIRECT_OAUTH_URL | The link of your ngrok with the oAuth route              | `string` |
+
+Now then we setup the `.env` file in the `example` folder follow the next commands.
+
+```bash
+# Go to the example folder
+$ cd example
+
+# Install the dependencies
+$ npm run install
```
Comment (onneri) [inline INSTALLATION.md]: npm install without "run"

File: INSTALLATION.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# **Example Installation**
```
Comment (onneri) [inline INSTALLATION.md]: README.md would be better

Comment (onneri) [reply INSTALLATION.md]: requirements:
NODE version
ngork
instructions about ngrok
etc...

File: example/.env.example [hunk 3]
```diff
@@ -0,0 +1,4 @@
+SLACK_CLIENT_SECRET=
+VITE_SLACK_CLIENT_ID=
+VITE_REDIRECT_URL=
+VITE_REDIRECT_OAUTH_URL=
```
Comment (onneri) [inline example/.env.example]: the las one is not redirect url

File: example/frontend/App.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import SlackLogin from 'react-slack-login'
+import './App.css'
+
+function App() {
+
+  const url = '<YOUR_REDIRECT_URL>'
```
Comment (onneri) [inline example/frontend/App.jsx]: this file is not used, not required

**Comment by hectorbenitez** [comment]
please remove .DS_Store file

**Pull request merged**